src/App.tsx

```diff
@@ -1,14 +1,66 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { Filters } from './types/Filters';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState<string>('');
+  const [status, setStatus] = useState<string>('all');
+  const [selectedTodoId, setSelectedTodoId] = useState<number | null>(null);
+
+  const [loadingTodos, setLoadingTodos] = useState(true);
```

Comment:
for boolean variables use prefixes (`is `, `has `)
```suggestion
  const [isLoadingTodos, setIsLoadingTodos] = useState(true);
```